test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json

```diff
@@ -36,5 +38,8 @@
   "compilationOptions": {
     "allowUnsafe": true
   },
-  "testRunner": "xunit"
+  "testRunner": "xunit",
+  "content": [
+    "TestResources/testCert.pfx"
```

Comment:
It expires 9/15/2016 at 11:21PM. Our test `validationCallback`s always return true, so it might not matter. The test is already issued by "localhost".